covid_model/analysis/run_compartment_report.py

```diff
@@ -37,4 +42,25 @@
             ax.legend(loc='upper left')
 
     fig.tight_layout()
-    plt.show()
+    if args.save_prefix:
```

Comment:
added optional saving of figs, and added non-normalized version of the same plots.